### ISSUE TITLE
Suppress equation editor

### ISSUE THREAD

**Issue by mackensen** [CLAMP-IT/clampmail#51]
The equation editor will never render correctly in an email, so we should consider removing the option altogether. Per [MDL-52996](https://tracker.moodle.org/browse/MDL-52996) and [Editor API](https://docs.moodle.org/dev/Editor_API) this should be possible.

**Comment by azito122** [comment]
At the moment this is blocked by https://tracker.moodle.org/browse/MDL-64512

**Comment by kwiliarty** [comment]
I just encountered something today that might be related. Using current LAE 3.8: Sending a youtube link through clampmail gives a bad result in HTML mail, I believe because the Multimedia plugins filter and hitting the link and trying to build a player around it. Turning off the filter allows the email to work correctly. I wonder: should we be preventing filters from applying to the contents of an email? I realize this might be a separate issue.
